libs/ledgerjs/packages/context-module/src/loaders/TokenLoader.ts

```diff
@@ -0,0 +1,11 @@
+import { LoaderOptions } from "../models/LoaderOptions";
+import { Transaction } from "../models/Transaction";
+import { ContextLoader } from "./ContextLoader";
+
+export class CalMetadataFetcher implements ContextLoader {
```

Comment:
[SHOULD] File name and classe name should be similar

Reply:
Already fixed on the branch feat/dsdk-284-context-module-cal but will move it before 👍